Assets/HoloToolkit-Preview/QRTracker/Plugins/QRCodesTrackerPlugin.Placeholder.dll.mdb.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
+guid: 9b91909175255dd40bbabf1f29de0b2e
```

Comment:
Is this dll to be replaced by direct windows api calls?

Reply:
eventually, when it is no longer a preview / experimental feature

Reply:
This API is do nothing for non UWP and Editor. Unity links against this dll for that purpose.